code/datums/entities/player_sticky_ban.dm

```diff
@@ -131,3 +131,114 @@ BSQL_PROTECT_DATUM(/datum/entity/stickyban)
 		"ip",
 		"linked_stickyban",
 	)
+
+/proc/get_player_is_alt(ckey)
```

Comment:
Could use a rename or atleast autodoc since it actually returns a list of alt ckeys.